client/src/components/MenuBar.js

```diff
@@ -3,8 +3,12 @@ import Nav from "react-bootstrap/Nav";
 import Navbar from "react-bootstrap/Navbar";
 import NavDropdown from "react-bootstrap/NavDropdown";
 import "./MenuBar.css";
-
 export const MenuBar = () => {
+	const scroll = "book";
+	const handleClick = () => {
+		document.getElementById(scroll).scrollIntoView({ behaviour: "smooth" });
+	};
```

Comment:
This seems to introduce a bit of complexity in your page, won't it be simpler to just navigate to the booking page from the link?

Reply:
Thats the issue I had.. It navigates to the top of the page and the admission information is there.. so both links in the screening dropdown will take you to the top which has admission info and you would still need to scroll down to get to the bookings form